src/nix/daemon.cc

```diff
@@ -273,8 +273,12 @@ static std::pair<TrustedFlag, std::string> authPeer(const PeerInfo & peer)
 /**
  * Run a server. The loop opens a socket and accepts new connections from that
  * socket.
+ *
+ * @param trustClientOpt If present, force trusting or not trusted the client.
+ * Otherwise, decide based on the authentication settings and user credentials
+ * (from the unix domain socket).
  */
-static void daemonLoop()
+static void daemonLoop(std::optional<TrustedFlag> trustClientOpt)
```

Comment:
I think `s/trustClientOpt/forceTrustClientOpt` may be nice.

Reply:
Done!